code/modules/client/preference/preferences_mysql.dm

```diff
@@ -261,21 +260,19 @@
 		player_alt_titles = params2list(query.item[58])
 		organ_data = params2list(query.item[59])
 		rlimb_data = params2list(query.item[60])
-		nanotrasen_relation = query.item[61]
-		speciesprefs = text2num(query.item[62])
+		speciesprefs = text2num(query.item[61])
```

Comment:
Existing database entries aren't going to match the new indexes for species prefs onward unless the entire column gets removed from all records in the database... I suppose that really all depends on how much attention the person making the database changes pays...

Afterall, knee-high socks aren't an acceptable species.